.github/draft-release-notes-config.yml

```diff
@@ -16,25 +16,28 @@ replacers:
 categories:
   - title: 'Breaking Changes'
     labels:
-      - 'Breaking Changes'
+      - 'breaking changes'
+  - title: 'Security'
+    labels:
+      - 'security fix'
   - title: 'Features'
     labels:
-      - 'Features'
+      - 'features'
```

Comment:
'features' -> `feature`

Reply:
committed in @mingshl
[singular form in label name](https://github.com/opensearch-project/search-processor/pull/84/commits/38ef698f536edf5387cf681a9b273bb4ebae0b7c)